docs/api.asciidoc

```diff
@@ -51,12 +51,12 @@ The given `context` argument must be an object and can contain any property that
 
 
 [float]
-[[apm-set-tags]]
```

Comment:
I have checked it and could not find any reference in our docs

Reply:
@bmorelli25 , is there anyway to make sure `apm-set-tags` reference is not used outside of our documentation?

Reply:
Kind of. I have _almost_ every elastic repo cloned locally, so I just `ctrl+f` the id before removing/changing it. The real way to test is to remove the id, then run a `$GIT_HOME/docs/build_docs.pl --all` to build all elastic documentation locally. The problem is this builds from the remote instead of local changes...

But! We have a new documentation process for when IDs are removed/changed to catch any problems. You'll need to add a file to this PR: `redirects.asciidoc`. This file will be hidden from your documentation, but will set up a landing page in case any links still use the old ID. Here are the contents of this file:
```
["appendix",role="exclude",id="redirects"]
= Deleted pages

The following pages have moved or been deleted.

[role="exclude",id="apm-set-tags"]
=== `apm.setTags()`

This page has moved. Please see <<apm-add-tags>> instead.
```

Check out the `redirects` file in `apm-server` if you'd like:
https://github.com/elastic/apm-server/blob/master/docs/redirects.asciidoc

Reply:
Thanks @bmorelli25 I will do the changes as suggested :)